### ISSUE TITLE
docker arm64 run error ,because arm64 missed lchown syscall 

### ISSUE THREAD

**Issue by jefby** [docker/docker#15079]
Hello everyone,
```
docker info: 
Client version: 1.7.1-rc3
Client API version: 1.19
Go version (client): devel +cc8f544 Fri Jul 17 03:34:11 2015 +0000
Git commit (client): 47d5df0-dirty
OS/Arch (client): linux/arm64
Server version: 1.7.1-rc3
Server API version: 1.19
Go version (server): devel +cc8f544 Fri Jul 17 03:34:11 2015 +0000
Git commit (server): 47d5df0-dirty
OS/Arch (server): linux/arm64
uname -a: linux-3.19.8
```

I want to run docker1.7.1-rc3 in arm64 machine,and now it can't work now,when i run command:

docker pull fedora

it will exit  and print error log like follow:

```
ded7cd95e059: Error pulling image (latest) from fedora, ApplyLayer exit status 1 stdout:  stderr: lchown /bin: function not implemented
48ecf305d2cf: Download complete
Error pulling image (latest) from fedora, ApplyLayer exit status 1 stdout:  stderr: lchown /bin: function not implemented
```

why ? and how to fix it ??

**Comment by hqhq** [comment]
Where did you get the Docker running on ARM64 with version 1.7.1-rc3? I think we have problems compile Docker on ARM64 for that version.

**Comment by jefby** [comment]
@hqhq Hi，i git clone offical docker repo,and compile it ,and now it can work ,but not very well.

I don't know how to fix the lchown function error,so if you know why please tell me,thanks very much!

**Comment by hqhq** [comment]
@jefby Sorry for the late, just picked up the ARM64 stuff, are you still seeing this error? I can't reproduce it on the latest Docker.

**Comment by jefby** [comment]
@hqhq I have report this bug to go community，and they fixed this in go 1.5-beta3 ,so it's ok now.
